scheduler/scheduler.go

```diff
@@ -189,6 +189,14 @@ func (s *Scheduler) Sync(ctx context.Context, client schema.ClientMeta, tables s
 
 	// send migrate messages first
 	for _, table := range tables.FlattenTables() {
+		if syncClient.deterministicCQID {
+			for i, c := range table.Columns {
+				if c.PrimaryKey && c.Name != schema.CqIDColumn.Name {
+					table.Columns[i].PrimaryKey = false
+				}
+			}
+		}
```

Comment:
If Deterministic_CQ_ID is enabled, then that should be the only PK that is being used. Update the Table definition prior to sending the SyncMessage